changelog/0.37.md

```diff
@@ -0,0 +1,89 @@
+
+## v0.37.0
+
+Released 2024-04-03
+
+> [!WARNING]
+> **Security Notice(s)**
+>
+> - This upgrade mitigates the following medium security vulnerabilities:
+>   - [CVE-2023-6152](https://github.com/advisories/GHSA-3hv4-r2fm-h27f)
+>   - [CVE-2024-1442](https://github.com/advisories/GHSA-5mxf-42f5-j782)
+<!-- -->
+> [!IMPORTANT]
+> **Platform Administrator Notice(s)**
+>
+> - When validating config against the schema you may encounter cases where they disagree on what type a field should be and it may be necessary to tweak either the config or the schema to match the intent.
+> - There is now an additional `CK8S_FLAVOR` called `air-gapped` with similar settings to `prod` but with some additional configuration to avoid any outgoing traffic to the Internet.
+> - Capacity management has been reworked with new alerts and a Grafana dashboard. In order for these to work, nodes must be labeled with `elastisys.io/node-group=<node-group>`.
+> - Local requirements have been updated, run `./bin/ck8s install-requirements` to update them.
+> - The `openstack-monitoring` chart will be installed by default if you are running Kubernetes on an Openstack environment (`.openstackMonitoring.enabled = true` in `defaults/common-config.yaml`).
+>   You need to make sure that `openstack-cloud-controller-manager` can expose metrics by setting adding the `--bind-address=0.0.0.0` flag.
+> - The image used for alertmanager can now be customized by setting `.prometheus.alertmanagerSpec.image = {registry: "...", repository: "...", tag: "..."}` for sc, or `.user.alertmanager.image = "registry/repository:tag"` for wc.
+> - In this version some of names used for falco rules and macros changed, and also the method for appending or override existing rules changed. If you have falco exception in the override configs you will need to update them.
+>   You can find more information about the new rules names, [here](https://github.com/falcosecurity/rules/tree/falco-rules-3.0.1/rules), and about the new selective rules overrides, [here](https://falco.org/docs/rules/overriding/).
+> - Added a new diagnostics command with the primary goal of aiding in supporting self-managed customers. By default it encrypts the file with the sops config inside of `$CK8S_CONFIG_PATH` but setting `$CK8S_PGP_FP` will override this.
+> - The `matchConditions` feature in the HNC webhook is dependent on Kubernetes v1.28+ and cert-manager v1.12+ and will not work with older versions.
+> - If you want to manage the organization roles of your users from within Grafana user you will need to enable `.grafana.user.oidc.skipRoleSync` in `sc-config.yaml`
+<!-- -->
+> [!NOTE]
+> **Application Developer Notice(s)**
+>
+> - Cert-manager Issuers and ClusterIssuers have been updated to use the [recommended](https://cert-manager.io/docs/configuration/acme/http01/#ingressclassname) `ingressClassName` field over the `class` field. We recommend that Application Developers use the `ingressClassName` field as well.
+
```

Comment:
Could you add this thing to the welcome dashboard as well? Since it includes a recommendation to app devs.

Reply:
done